src/core/curlify.js

```diff
@@ -25,15 +25,18 @@ export default function curl( request ){
   if ( headers && headers.size ) {
     for( let p of request.get("headers").entries() ){
       let [ h,v ] = p
-      type = v
       curlified.push( "-H " )
       curlified.push( `"${h}: ${v}"` )
+      if (isMultipartFormDataRequest) {
+        break
+      } else {
+        isMultipartFormDataRequest = /content-type/i.test(h) && /multipart\/form-data/i.test(v)
```

Comment:
Actually I did a mistake in regex definition here
```suggestion
        isMultipartFormDataRequest = /^content-type$/i.test(h) && /^multipart\/form-data$/i.test(v)
```
Regex should be fined as exact match